metadata.yaml

```diff
@@ -17,10 +17,10 @@ maintainers:
     - Luca Bello <luca.bello@canonical.com>
     - Simon Aronsson <simon.aronsson@canonical.com>
 
-#docs: https://discourse.charmhub.io/t/grafana-agent-k8s-docs-index/5605
+docs: https://github.com/canonical/grafana-agent-operator/blob/main/README.md
```

Comment:
- When there is no `docs` key, charmhub automatically renderes the readme.
- Currently there are no discourse docs so the `docs` key should be omitted.

Reply:
Ha, you're right. Alrighty, let's give this a try.